js/test/units/testunitfmt_gl_ES.js

```diff
@@ -0,0 +1,331 @@
+        test.expect(1);
+        var m1 = MeasurementFactory({
+            unit: "square centimeter",
+            amount: 2
+        });
+
+        var uf = new UnitFmt({
+            locale: "gl-ES", 
+            autoConvert: true,
+            length: "long"
+        });
+
+        var str = uf.format(m1);
+        test.equal(str, "2 centímetros cadrados");
+        test.done();
+    },
+    testUnitFormatArea3_gl_ES: function(test) {
+        test.expect(1);
+        var m1 = MeasurementFactory({
+            unit: "hectare",
+            amount: 1000
+        });
+
+        var uf = new UnitFmt({
+            locale: "gl-ES", 
+            autoConvert: true,
+            length: "short"
+        });
+
+        var str = uf.format(m1);
+        test.equal(str, "10 km²");
+        test.done();
+    },
+    testUnitFormatArea4_gl_ES: function(test) {
+        test.expect(1);
+        var m1 = MeasurementFactory({
+            unit: "hectare",
+            amount: 1000
+        });
+
+        var uf = new UnitFmt({
+            locale: "gl-ES", 
+            autoConvert: true,
+            length: "long"
+        });
+
+        var str = uf.format(m1);
+        test.equal(str, "10 quilómetros cadrados");
+        test.done();
+    },
+    testUnitFormatFuelConsumption1_gl_ES: function(test) {
+        test.expect(1);
+        var m1 = MeasurementFactory({
+            unit: "km/liter",
+            amount: 2
+        });
+
+        var uf = new UnitFmt({
+            locale: "gl-ES",
+            autoConvert: true,
+            length: "short"
+        });
+        var str = uf.format(m1);
+        test.equal(str, "2 km/l");
+        test.done();
+    },
+    testUnitFormatFuelConsumption2_gl_ES: function(test) {
+        test.expect(1);
+        var m1 = MeasurementFactory({
+            unit: "km/liter",
+            amount: 2
+        });
+
+        var uf = new UnitFmt({
+            locale: "gl-ES",
+            autoConvert: true,
+            length: "long"
+        });
+        var str = uf.format(m1);
+        test.equal(str, "2 quilómetros por litro");
+        test.done();
+    },
+    testUnitFormatLength1_gl_ES: function(test) {
+        test.expect(1);
+        var m1 = MeasurementFactory({
+            unit: "mile",
+            amount: 2000
+        });
+
+        var uf = new UnitFmt({
+            locale: "gl-ES",
+            autoConvert: true,
+            length: "short"
+        });
+        var str = uf.format(m1);
+        test.equal(str, "3,21868 Mm");
+        test.done();
+    },
+    testUnitFormatLength2_gl_ES: function(test) {
+        test.expect(1);
+        var m1 = MeasurementFactory({
+            unit: "mile",
+            amount: 2000
+        });
+
+        var uf = new UnitFmt({
+            locale: "gl-ES",
+            autoConvert: true,
+            length: "long"
+        });
+        var str = uf.format(m1);
+        test.equal(str, "3,21868 megameters");
```

Comment:
Again, this one is in English. Perhaps test some other measure than megameters that does have a translation in Galician?